### DISCUSSION TITLE
blurring distant map areas

### DISCUSSION THREAD

**Discussion by wiesehahn** [maplibre/maplibre-gl-js#1288] [General]
I wonder if it would be possible and make sense to add an option for a blurring effect applied on the map by distance?

This might add some visual depth to the map when inclined, similar to a tilt/shift effect. On the other hand it might improve the visual appearance when tiles of lower zoom levels are loaded in the distance (e.g. with Terrain3D).

**Comment by wipfli** [comment]
Oh that would look nice I guess

**Reply by wiesehahn** [reply]
would be necessary to get a depth map to work with distance related styling (https://github.com/maplibre/maplibre-gl-js/discussions/1777)
